class.jetpack.php

```diff
@@ -1607,7 +1607,7 @@ public static function load_modules() {
 				continue;
 			}
 
-			if ( ! @include( Jetpack::get_module_path( $module ) ) ) {
+			if ( ! include_once( Jetpack::get_module_path( $module ) ) ) {
```

Comment:
"[if the code from a file has already been included, it will not be included again, and include_once returns TRUE.](http://php.net/manual/en/function.include-once.php)"